pydm/widgets/archiver_time_plot.py

```diff
@@ -403,3 +474,19 @@ def setCurves(self, new_list: List[str]) -> None:
             )
 
     curves = Property("QStringList", getCurves, setCurves, designable=False)
+
+
+class PyDMDateAxisItem(DateAxisItem):
```

Comment:
I discovered in more testing that my PyDMDateAxisItem might not actually be necessary. I initially added it to capture user mouse interaction on the X-Axis, but this can be captured in other ways: `PyDMArchiverTimePlot.plotItem.vb.sigRangeChangedManually`
is a signal that emits when the user changes the ViewBox and on which axis